### DISCUSSION TITLE
요즘 Docker 공부하신다고 들었는데

### DISCUSSION THREAD

**Discussion by simchodi** [guui-programmers/Programmers-Algorithm#22] [Q&A] [answered]
요즘 Docker 공부하신다고 들었는데 간단하게 설명글 한번 부탁드릴게요

**Comment by thyoondev** [comment, marked as answer]
제가 이전에 작성한 포스트에서 인사이트를 얻으셨으면 좋겠습니다.
[왜 도커(Docker)가 필요한가?](https://velog.io/@thyoondev/%EC%99%9C-%EB%8F%84%EC%BB%A4Docker%EA%B0%80-%ED%95%84%EC%9A%94%ED%95%9C%EA%B0%80)

**Comment by HyunDel** [comment]
도커는 애플리케이션을 신속하게 구축, 테스트 및 배포할 수 있는 소프트웨어 플랫폼이예요 !!
